guardian/prosemirror-invisibles
# Add vite, prosemirror-dev-tools

## What does this change?

Brings the developer experience on this repo up to scratch, swapping `rollup` for `vite`, and adding `prosemirror-dev-tools`.

## How to test

Checkout this branch, and run `yarn`. Try `yarn watch`. Does the project start and work correctly? Can you see the prosemirror dev tools in the bottom rhs doing its thing?
